components/ClaimStatus.tsx

```diff
@@ -23,10 +29,18 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ loading = false, headi
       </div>
       <div className="status-box">
         <div className="topbar">
-          <h3 className="next-steps">{t('claim-status.next-steps')}</h3>
+          <h3 className="next-steps">{t('claim-status.your-next-steps')}</h3>
         </div>
         <div className="explanation">
-          <NextSteps loading={loading} nextSteps={nextSteps} />
+          <NextSteps loading={loading} nextSteps={yourNextSteps} />
+        </div>
+      </div>
+      <div className="status-box">
+        <div className="topbar">
+          <h3 className="next-steps">{t('claim-status.edd-next-steps')}</h3>
+        </div>
+        <div className="explanation">
+          <NextSteps loading={loading} nextSteps={eddNextSteps} />
```

Comment:
@lomky Again, I just tossed all of this in here and assumed you'd do some finessing.

Reply:
oh this will be a fun merge in! 😅

Reply:
quick guidance since I refactored a component under ya:

A lot of the structure here got nixed:
 - the `h3, `status-box`, `explanation` & `topbar` gone

now you just call `NextSteps` and pass it the next steps and header

So i _think_ this component return should end up just looking like:

```tsx
    <div className="claim-status claim-section">                                                                                        
      <h2>{t('claim-status.title')}</h2>                                                                                                
      <div className="pending-status claim-subsection">                                                                                 
        <TextLine loading={loading} header text={t(heading)} />                                                                         
      </div>                                                                                                                            
      <div className="summary">                                                                                                         
        <TransLine loading={loading} i18nKey={summary.i18nKey} links={summary.links} />                                                 
      </div>                                                                                                                            
      <NextSteps loading={loading} header={t('claim-status.your-next-steps')} nextSteps={yourNextSteps} />                                       
      <NextSteps loading={loading} header={t('claim-status.edd-next-steps')} nextSteps={eddNextSteps} />                                       
    </div>  
```